website/docs/r/storage_bucket.markdown

```diff
@@ -101,6 +125,22 @@ The `CORS` object supports the following:
 
 * `max_age_seconds` (Optional) Specifies time in seconds that browser can cache the response for a preflight request.
 
+* `server_side_encryption_configuration` (Optional) A configuration of server-side encryption configuration (documented below)
```

Comment:
Just "A configuration of server-side encryption for the bucket"

Reply:
fixed